.gitignore

```diff
@@ -0,0 +1,4 @@
+draft-*.txt
+draft-*.html
+draft-.*.xml
```

Comment:
```suggestion
draft-*.xml
```

Reply:
Good catch, I added this one last minute but it usually get `rm`ed when running `make`. I actually did not grasp which part of the Makefile does this....